{"name": "Проверка неуспешного логина с некорректным паролем", "status": "passed", "steps": [{"name": "Выполняем логин от имени курьера", "status": "passed", "parameters": [{"name": "login", "value": "'gxweffqkko'"}, {"name": "password", "value": "'pflskcujqt'"}], "start": 1710018471190, "stop": 1710018471628}], "start": 1710018471190, "stop": 1710018471629, "uuid": "ba5bcd54-bd02-4ee9-ba97-973ac030ea45", "historyId": "2b1ea438a8d9d7d96cca7a21db237e9b", "testCaseId": "2b1ea438a8d9d7d96cca7a21db237e9b", "fullName": "tests.test_courier_login.TestCourierLogin#test_loggining_incorrect_password_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "pavel"}, {"name": "thread", "value": "18992-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}